lib/conrad/emitters/kinesis.rb

```diff
@@ -0,0 +1,79 @@
+require 'conrad/errors'
+
+module Conrad
+  # A module containing all of conrad's built in event emitters for outputting
+  # events
+  module Emitters
+    # Basic emitter for sending events to AWS's kinesis event stream. The emitter will
+    # attempt to use values configured in the running environment according to
+    # the AWS SDK documentation (such as from ~/.aws/credentials).
+    class Kinesis
+      # Error for responding with issues around kinesis credential creation
+      class InvalidAwsCredentials < ::Conrad::Error
```

Comment:
How would you feel about moving this into the `errors` file to keep all the errors in one place?

Reply:
Yep sounds good